src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -1,32 +1,41 @@
 <Project>
   <PropertyGroup>
-    <BlazorLinkOnBuild Condition="$(BlazorLinkOnBuild) == ''">true</BlazorLinkOnBuild>
+    <BlazorWebAssemblyEnableLinking Condition="$(BlazorWebAssemblyEnableLinking) == ''">true</BlazorWebAssemblyEnableLinking>
   </PropertyGroup>
 
   <Target
-    Name="PrepareBlazorOutputs"
+    Name="_PrepareBlazorOutputs"
     DependsOnTargets="_ResolveBlazorInputs;_ResolveBlazorOutputs;_GenerateBlazorBootJson">
   </Target>
 
   <Target Name="_ResolveBlazorInputs" DependsOnTargets="ResolveReferences;ResolveRuntimePackAssets">
+    <Warning Text="BlazorLinkOnBuild has been renamed to BlazorWebAssemblyEnableLinking. Please update your project files to use the new property."
```

Comment:
I've added a warning for this so we have a path of migration for users using this flag. It's not easy to figure out that an MSBuild property basically no-oped.

Reply:
I would also be fine with failing the build if `BlazorLinkOnBuild != ''`.